README.md

```diff
@@ -24,7 +24,7 @@ To spin up the site, we need two steps:
 
 (1) We need to update the backend to get the data. `cd scrapers` then run `python update.py`.
 
-(2) We then can update the frontend, via running `npm start`. This will open a browser tab that updates live as you edit code.
+(2) We then can update the frontend, via running `npm run dev`. This will start a developer server. Open a browser tab to [`http://localhost:5173/`](http://localhost:5173/), which will update live as you edit code.
```

Comment:
Why port 5173 and not 3000?

Reply:
It's the default, but [its configurable.](https://vite.dev/config/server-options.html#server-port).